<?xml version="1.0" ?>
<robot name="ropod">
  <link name="ropod__base_link">
    <inertial>
      <mass value="0.01"/>
      <origin rpy="0  0  0" xyz="0  0  0"/>
      <inertia ixx="0.0" ixy="0.0" ixz="0.0" iyy="8.383E-4" iyz="0.0" izz="1.66E-3"/>
    </inertial>
    <collision name="ropod__base_link_collision">
      <origin rpy="0  0  0" xyz="0    0    0.15"/>
      <geometry>
        <box size="0.6 0.6 0.1"/>
      </geometry>
    </collision>
    <collision name="ropod__base_link_collision_left_rear_wheel">
      <origin rpy="0  0  0" xyz="-0.25  0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </collision>
    <collision name="ropod__base_link_collision_left_top_wheel">
      <origin rpy="0  0  0" xyz="0.25  0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </collision>
    <collision name="ropod__base_link_collision_right_top_wheel">
      <origin rpy="0  0  0" xyz="0.25 -0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </collision>
    <collision name="ropod__base_link_collision_right_rear_wheel">
      <origin rpy="0  0  0" xyz="-0.25 -0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </collision>
    <visual name="ropod__base_link_visual">
      <origin rpy="0  0  0" xyz="0    0    0.15"/>
      <geometry>
        <box size="0.6 0.6 0.1"/>
      </geometry>
      <material
        name="">
        <color
          rgba="0.35 0.35 0.35 1" />
      </material>
    </visual>
    <visual name="ropod__base_link_visual_left_rear_wheel">
      <origin rpy="0  0  0" xyz="-0.25  0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </visual>
    <visual name="ropod__base_link_visual_left_top_wheel">
      <origin rpy="0  0  0" xyz="0.25  0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </visual>
    <visual name="ropod__base_link_visual_right_top_wheel">
      <origin rpy="0  0  0" xyz="0.25 -0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </visual>
    <visual name="ropod__base_link_visual_right_rear_wheel">
      <origin rpy="0  0  0" xyz="-0.25 -0.25  0.05"/>
      <geometry>
        <sphere radius="0.05"/>
      </geometry>
    </visual>
    <visual name="ropod__base_link_visual_laser">
      <origin rpy="0  0  0" xyz="0.28499  0       0.22"/>
      <geometry>
        <box size="0.04 0.04 0.04"/>
      </geometry>
    </visual>
    <visual name="ropod__base_link_visual_lasertop">
      <origin rpy="0  0  0" xyz="0.28499  0       0.265"/>
      <geometry>
        <box size="0.04 0.04 0.01"/>
      </geometry>
    </visual>
  </link>
</robot>
